packages/devtools_app/test/table_test.dart

```diff
@@ -179,6 +184,114 @@ void main() {
         ),
       ));
       expect(find.byWidget(table), findsOneWidget);
+      // TODO(jacobr): add a golden image test.
+    });
+
+    testWidgets('displays with wide column', (WidgetTester tester) async {
+      final table = FlatTable<TestData>(
+        columns: [
+          flatNameColumn,
+          _NumberColumn(),
+          _WideColumn(),
+        ],
+        data: flatData,
+        onItemSelected: noop,
+        keyFactory: (data) => Key(data.name),
+        sortColumn: flatNameColumn,
+        sortDirection: SortDirection.ascending,
+      );
+      await tester.pumpWidget(wrap(
+        SizedBox(
+          width: 800.0,
+          height: 200.0,
+          child: table,
+        ),
+      ));
+      expect(find.byWidget(table), findsOneWidget);
+      {
+        final FlatTableState<TestData> state =
+            tester.state(find.byWidget(table));
+        final columnWidths = state.computeColumnWidths(800.0);
+        expect(columnWidths.length, equals(3));
+        expect(columnWidths[0], equals(300.0));
+        expect(columnWidths[1], equals(400.0));
+        expect(columnWidths[2], equals(36.0));
+      }
+
+      // TODO(jacobr): add a golden image test.
+
+      await tester.pumpWidget(wrap(
+        SizedBox(
+          width: 200.0,
+          height: 200.0,
+          child: table,
+        ),
+      ));
+
+      {
+        final FlatTableState<TestData> state =
+            tester.state(find.byWidget(table));
+        final columnWidths = state.computeColumnWidths(200.0);
+        expect(columnWidths.length, equals(3));
+        expect(columnWidths[0], equals(300.0)); // Fixed width column.
+        expect(columnWidths[1], equals(400.0)); // Fixed width column.
+        expect(columnWidths[2], equals(0.0)); // Variable width column.
+      }
+
+      // TODO(jacobr): add a golden image test.
```

Comment:
I added some golden tests but the dimensions of the SizedBox aren't being respected in the golden screenshot and I don't have time to figure out why.